.zuul.yaml

```diff
@@ -1,16 +1,20 @@
 ---
 - job:
-    name: functional-autoscaling-tests-osp18
+    name: functional-autoscaling-metrics-tests-osp18
```

Comment:
You can't just rename this, it is used in the telemetry-operator repo.

I suggest keeping the name as `functional-autoscaling-tests-osp18` for now, even though the scope of the job is extended. This will keep things working in telemetry-operator.

Alternatively, create a new job that has all the tests in it, that will get added to.
(This could be called something like `feature-verification-tests-osp18`).
This keeps the consolidation of jobs cleaner, and reduces the issues like you're seeing with the job not being in telemetry-operator.